CONTRIBUTING.adoc

```diff
@@ -60,6 +54,91 @@ them before we can merge your contribution.
 don’t add references or links to things you or your employer have
 created as others will do so if they appreciate it.
 
+== Getting set up for local development
+
+Before you begin contributing, its important to get your local machine set up to run and test the site. For convenience, we provide a link:container-images/local-dev/[container image] that contains all of the development tools. The simplest way to run the image is as follows:
+
+....
+docker run quay.io/redhat-cop/uncontained
```

Comment:
mwitzenm@surfy ~/src/uncontained.io (esauer/improve-local-dev) $ docker run quay.io/redhat-cop/uncontained
Unable to find image 'quay.io/redhat-cop/uncontained:latest' locally
Trying to pull repository quay.io/redhat-cop/uncontained ... 
/usr/bin/docker-current: unauthorized: access to the requested resource is not authorized.
See '/usr/bin/docker-current run --help'.

Reply:
@mwitzenm yes, you'll have to build the image yourself for now. It won't build/push to quay until we merge.

Reply:
adjust the Dockerfile here container-images/local-dev/Dockerfile for the local-dev directory instead of this directory local-builder
then build the image
docker build -f container-images/local-dev/Dockerfile -t uncontained-local-dev .
then run the image
docker run -u $(id -u) --net=host -v <PATH-TO->uncontained.io:/uncontained.io:Z -it uncontained-local-dev